AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsViewModel.swift

```diff
@@ -10,32 +10,33 @@ class ThresholdSettingsViewModel: ObservableObject {
     @Published var thresholdMedium = ""
     @Published var thresholdHigh = ""
     @Published var thresholdVeryHigh = ""
-    let initialThresholds: [Int32]
-    
-    init(initialThresholds: [Int32]) {
+    let initialThresholds: ThresholdsValue
+
+    init(initialThresholds: ThresholdsValue) {
         self.initialThresholds = initialThresholds
     }
+
+    func resetToDefault() -> ThresholdsValue {
+         return initialThresholds
+     }
```

Comment:
[SUGESTION ONLY] What do you think about making it one liner:  `func resetToDefault() -> ThresholdsValue { initialThresholds }` ?

Reply:
**Sure!** Always happy to make the code prettier :D